spyder_line_profiler/spyder/widgets.py

```diff
@@ -68,7 +56,27 @@
 
 WEBSITE_URL = 'http://pythonhosted.org/line_profiler/'
 
+locale_codec = QTextCodec.codecForLocale()
+
+
+# This is needed for testing this module as a stand alone script
+try:
+    _ = get_translation("line_profiler", dirname="spyder_line_profiler")
+except KeyError as error:
+    import gettext
+    _ = gettext.gettext
```

Comment:
Could you explain a little bit more why this is needed please? Is this related with the manually added `.pot` file?

Reply:
No idea, I copied this line from the previous version